modules/installing-rhv-requirements.adoc

```diff
@@ -30,6 +30,9 @@ If you increase the number of virtual machines in the {rh-virtualization} enviro
 *** 16 GiB or more for each of the three control plane machines which provide the control plane.
 *** 16 GiB or more for each of the three compute machines, which run the application workloads.
 * The {rh-virtualization} storage domain must meet link:https://access.redhat.com/solutions/4770281[these etcd backend performance requirements].
+* For affinity group support:
+** Production setup: For hard anti-affinity, you need a minimum of three physical machines. For more than three worker nodes, one physical machine per worker or control plane. Workers and control planes can be on the same physical machine. For example, if you have three workers and three control planes, you need three physical machines. If you have four workers and three control planes, you need four physical machines.
+** Non-production setup, such as a lab: You can define affinity groups with multiple workers or control planes on the same physical machine, but this setup does not guarantee redundancy.
```

Comment:
One more thing: for lab setups people should remove all affinity groups to enable colocating the machines on as few physical nodes as possible.

Reply:
@janoszen  OK, take a look at the updated text.